learn/calling-java-code-from-ballerina.md

```diff
@@ -983,6 +985,30 @@ public function main() {
 
 As you can see, you need to first construct an instance of the `ArrayDeque` class. The `arrayDequeObj` variable refers to an `ArrayDeque` object. Then, you need to pass this variable to both the `pop` and `push` functions because the corresponding Java methods are instance methods of the`ArrayDeque` class. Therefore, you need an instance of the `ArrayDeque` class in order to invoke its instance methods. You can think of the `arrayDequeObj` variable as the method receiver.
 
+
+#### Calling Methods Asynchronously
+
+Ballerina internally uses a fixed number of threads, therefore when calling Java method, it should return in a reasonable time frame in order to avoid starvation in Ballerina code execution.
```

Comment:
```suggestion
Ballerina internally uses a fixed number of threads. Therefore, when calling a Java method, it should return in a reasonable time frame in order to avoid starvation in the Ballerina code execution.
```